packages/api-fetch/src/index.js

```diff
@@ -68,16 +76,24 @@ function apiFetch( options ) {
 					throw invalidJsonError;
 				}
 
+				const responseClone = response.clone();
+
 				return response.json()
 					.catch( () => {
-						throw invalidJsonError;
+						return responseClone.text()
+							.then( ( text ) => {
```

Comment:
Aside: [Async/Await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) could make this nicer to read:

```js
.catch( async () => {
	const text = await responseClone.text();
	checkCloudflareError( text );
	throw invalidJsonError;
} );
```